docs/simple_switch.md

```diff
@@ -685,3 +685,8 @@ you have two actions for table `t` where one has the action
 `m.read(result_field2)`, or if both of those calls are in the same
 action.  All calls to the `read()` method for `m` must have the same
 result parameter where the result is written.
+
+### BMv2 `meter rate units` implementation notes
+
+The user enters the meter rate and burtsize in
```

Comment:
It sounds reasonable that the meter _rate_ is in units of bytes per microsec, but the units of "burst size" should be some kind of unit like bytes.  It has nothing to do with time.  I do not know if simple_switch uses bytes or some other unit like kilobytes.

Reply:
From the names of variables and some of the math done to convert units, it appears like perhaps the burst sizes are specified in multiples of 1000 bits (Kbits would be confusing to many people, I think, as it would often be interpreted as multiples of 1024 bits).

Reply:
Also, these are likely the units used when a meter has meter type equal to MeterType.bytes in v1model.

Does BMv2 support different behavior for meters that have a type equal to MeterType.packets ?

If so, what are the units of the meter rates and burst sizes for such meters?

Reply:
> It sounds reasonable that the meter _rate_ is in units of bytes per microsec, but the units of "burst size" should be some kind of unit like bytes. It has nothing to do with time. I do not know if simple_switch uses bytes or some other unit like kilobytes.

Sorry, I missed the burst_size in bytes.  I will change the doc.

Reply:
> From the names of variables and some of the math done to convert units, it appears like perhaps the burst sizes are specified in multiples of 1000 bits (Kbits would be confusing to many people, I think, as it would often be interpreted as multiples of 1024 bits).

Makes sense to use number of kilo octets.

Reply:
> Also, these are likely the units used when a meter has meter type equal to MeterType.bytes in v1model.
> 
> Does BMv2 support different behavior for meters that have a type equal to MeterType.packets ?
> 
> If so, what are the units of the meter rates and burst sizes for such meters?

pps and packets.